chia/data_layer/data_store.py

```diff
@@ -49,10 +50,11 @@ class DataStore:
 
     db: aiosqlite.Connection
     db_wrapper: DBWrapper
+    random: Random
 
     @classmethod
-    async def create(cls, db_wrapper: DBWrapper) -> "DataStore":
-        self = cls(db=db_wrapper.db, db_wrapper=db_wrapper)
+    async def create(cls, db_wrapper: DBWrapper, random: Random) -> "DataStore":
```

Comment:
I'm feeling a bit funny about the data store needing randomness.  I understand and appreciate how this control is provided so we can get consistency in tests, but perhaps it can be given a default so it only need be specified when that control is wanted?  But also, is randomness critical?  If so, could it live outside the store and let the store just provide access to the stored data?